### DISCUSSION TITLE
How do I get the aws_config to load environment variables such as AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY?

### DISCUSSION THREAD

**Discussion by spencerbart** [awslabs/aws-sdk-rust#985] [Q&A]
I cannot get aws_config::load_from_env() to load environment variables. It keeps loading .aws/credentials and not the environment variables.

I make sure those credentials are present in the environment variables before I load aws_config but it still will not load them.

**Comment by spencerbart** [comment]
This is the code

```Rust
    println!("AWS_ACCESS_KEY_ID: {}", std::env::var("AWS_ACCESS_KEY_ID").unwrap());
    println!("AWS_SECRET_ACCESS_KEY: {}", std::env::var("AWS_SECRET_ACCESS_KEY").unwrap());

    let aws_config = aws_config::load_from_env().await;
    let ses_client = aws_sdk_sesv2::Client::new(&aws_config);
```

It crashes because it tries to load a non-existent file

**Reply by jdisanti** [reply]
I tried it myself, and it works. What version of the SDK did you observe this on?
